assets/files.wxs

```diff
@@ -1964,6 +1964,11 @@
             <File Id="fil6B026F6D5210498B99E5F14386F4F90B" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\Microsoft.PowerShell.ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 10 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Validate AccountManagement user cmdlets.Validate New-User.Can create New-User with only name
```powershell
Expected exactly 'TestUserNew', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 27
27:                 $result.Name | Should -BeExactly TestUserNew
```
Validate AccountManagement user cmdlets.Validate New-User.Can create New-User with password
```powershell
Expected exactly 'TestUserNew', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 38
38:                 $result.Name | Should -BeExactly TestUserNew
```
Validate AccountManagement user cmdlets.Validate New-User.Can create New-User with explicit properties in local machine
```powershell
Expected exactly 'samq2', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 54
54:                 $result.Name | Should -BeExactly "samq2"
```
Validate AccountManagement user cmdlets.Validate Remove-User.Can remove user account with pipeline
```powershell
Expected no exception to be thrown, but an exception "Cannot access a disposed object.
Object name: 'PrincipalContext'." was thrown from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1:134 char:35
    + ?               { $userIdentity | Remove-User -ErrorAction Stop } | Sho ?
    +                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 134
134:                 { $userIdentity | Remove-User -ErrorAction Stop } | Should -Not -Throw
```
Validate AccountManagement user cmdlets.Validate Remove-User.Can remove user account with parameter
```powershell
Expected no exception to be thrown, but an exception "Cannot bind argument to parameter 'Identity' because it is null." was thrown from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1:141 char:41
    +                 { Remove-User -Identity $userIdentity -ErrorAction St ?
    +                                         ~~~~~~~~~~~~~.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 141
141:                 { Remove-User -Identity $userIdentity -ErrorAction Stop } | Should -Not -Throw
```